### ISSUE TITLE
hash

### ISSUE THREAD

**Issue by brainmurder** [dolce/iziModal#27]
When you load the page with any hash, for example #test.
An error occurs:
**Uncaught TypeError: Cannot read property 'iziModal' of undefined**

```
  i.off("hashchange load").on("hashchange load", function(e) {
    var i = document.location.hash;
    if (0 === l)
      if ("" !== i) {
        t.each(t("." + o), function(e, n) {
          var o = t(n).iziModal("getState");
          "opened" != o && "opening" != o || "#" + t(n).attr("id") !== i && t(n).iziModal("close")
        });
        var n = t(i).data();
        "load" === e.type ? n.iziModal.options.autoOpen !== !1 && t(i).iziModal("open") : setTimeout(function() {
          t(i).iziModal("open")
        }, 200)
      } else
        t.each(t("." + o), function(e, i) {
          var n = t(i).iziModal("getState");
          "opened" != n && "opening" != n || t(i).iziModal("close")
        });
    else
      l = 0
  }),
```

The error in this line:

```
"load" === e.type ? n.iziModal.options.autoOpen !== !1 && t(i).iziModal("open") : setTimeout(function() {
          t(i).iziModal("open")
        }, 200)
```

The error is triggered even if iziModal not declared

**Comment by dolce** [comment]
I'll check, thank you.

**Comment by dolce** [comment]
@brainmurder, fixed and will be released in the next version (1.3.4). thanks
